css/main.css

```diff
@@ -0,0 +1,1485 @@
+* {
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+nav {
+  height: 80px;
+}
+
+.nav-1 {
+  display: flex;
+  width: 100%;
+  height: 31px;
+  background-color: black;
+  justify-content: space-between;
+  text-align: center;
+  padding-top: 8px;
+}
+
+.nav-1child {
+  padding-left: 20px;
+  padding-right: 20px;
+  justify-content: space-between;
+}
+
+.nav-1child a {
+  text-decoration: none;
+  padding-left: 5px;
+  padding-right: 5px;
+  color: rgb(172, 163, 163);
+  font-size: 13px;
+}
+
+.nav-2 {
+  display: flex;
+  background-color: white;
+  width: 100%;
+  height: 30px;
+  justify-content: space-between;
+  text-align: center;
+  padding-top: 10px;
+  border: 1px solid rgb(226, 224, 224);
+}
+
+.nav-2child {
+  padding-left: 20px;
+  padding-right: 20px;
+  justify-content: space-between;
+}
+
+.nav-2child a {
+  text-decoration: none;
+  padding-left: 5px;
+  padding-right: 5px;
+  color: rgb(85, 85, 85);
+}
+
+.nav-img {
+  padding-right: 20px;
+}
+
+.search-icon {
+  width: 20px;
+  height: auto;
+}
+
+/* main section */
+
+.section-1 {
+  display: grid;
+  height: 475px;
+  grid-template-columns: 2fr 1fr;
+  grid-template-rows: 1fr;
+  grid-column-gap: 5px;
+}
+
+.section-1-1 {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/kodak-796x448.jpg");
+  background-size: cover;
+  grid-gap: 2px;
+}
+
+.section1-1-heading {
+  font-size: 45px;
+  font-weight: 700;
+  letter-spacing: -0.84px;
+  color: #fff;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  gap: normal;
+  position: relative;
+  top: 40px;
+}
+
+.align-date {
+  text-align: right;
+  width: 100%;
+  height: 10px;
+  position: relative;
+  top: -40px;
+}
+
+.section1-1-heading-date {
+  color: #fff;
+  font-size: 19px;
+  font-weight: lighter;
+  margin-right: 11px;
+}
+
+.section1-1-heading-title-link {
+  text-decoration: none;
+  color: #fff;
+  position: relative;
+  top: 75px;
+  font-weight: lighter;
+  font-size: 23px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+.section1-flex-column {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  top: 175px;
+  padding: 5px;
+}
+
+.flexglue {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_